Classes/UI/HSMainListViewController.m

```diff
@@ -80,6 +80,18 @@ - (void)viewDidLoad
     [self refreshMyIssue];
     
     [self addCreditsToTable];
+    
```

Comment:
Why is a new UISearchController added? I am getting a crash after integrating this code. Am I missing something?